### ISSUE TITLE
cuDNN and CUDA Install Instructions on the Install.md Page Are Out of Date

### ISSUE THREAD

**Issue by ProGamerGov** [jcjohnson/neural-style#256]
Currently on the cuDNN section of Nvidia's website the following versions of cuDNN are listed:

> Download cuDNN v5.1 RC (June 19, 2016), for CUDA 8.0 RC
> 
> Download cuDNN v5.1 RC (June 16, 2016), for CUDA 7.5
> 
> Download cuDNN v5 (May 27, 2016), for CUDA 8.0 RC
> 
> Download cuDNN v5 (May 12, 2016), for CUDA 7.5

https://developer.nvidia.com/rdp/cudnn-download

The [Install.md](https://github.com/jcjohnson/neural-style/blob/master/INSTALL.md) page still uses Cuda 7.0 and Cudnn v4. 

---

In the latest newsletter/email cuDNN 5.1 RC is said to be "2.7x faster training of networks with 3x3 convolutions such as VGG".

**Comment by Runescaped** [comment]
I believe OP is aware of this, but just hasn't found the time to update it...

https://github.com/jcjohnson/neural-style/issues/245
https://github.com/jcjohnson/neural-style/issues/239

**Comment by thedonquixotic** [comment]
Yeah same problem here. I tried just running the code with 7.5 and 5.0 subbed in, but ran into some problems when trying to run the sudo cp cuda/lib64/libcudnn part. It says the directory doesn't exist. 

Any clue why? It says that usr/local/cuda-7.5/lib64 is not a directory. Looking at the directories cureated by the tar -xzvf part, I think it might be just called cuda, and forgoing the specific number in the directory title?

**Comment by Runescaped** [comment]
> It says that usr/local/cuda-7.5/lib64 is not a directory.

Are you typing in the correct command? There should be a backslash in front of usr.

`sudo cp cuda/lib64/libcudnn* **/**usr/local/cuda-7.0/lib64/`

**Comment by thorfinnx** [comment]
> It says that usr/local/cuda-7.5/lib64 is not a directory.

This might help you: https://devtalk.nvidia.com/default/topic/936212/cuda-setup-and-installation/tensorflow-cannot-find-cudnn-ubuntu-16-04-cuda7-5-/post/4880388/#4880388

**Comment by codesoon** [comment]
Anybody can write a new instruction?
